### DISCUSSION TITLE
Unlisted Scripts builds to exclude dependent packages, such as Vue and Element

### DISCUSSION THREAD

**Discussion by small-carbon** [wxt-dev/wxt#698] [Q&A]
Just defineUnlistedScript alone to exclude. Please tell me how to solve this problem. such as：
![image](https://github.com/wxt-dev/wxt/assets/887531/495bccb9-c152-49ff-ade6-cc74c1bb8ac8)

**Comment by aklinker1** [comment]
`exclude` does not exclude dependencies. It excludes the entire entrypoints when `--browser` matches a string in the array.

In your case, you're telling WXT to not build your content script when you run `wxt build --browser vue`.

If you want to exclude bundling a dependency, you need to mark it as "external" in vite. But I don't know why you would need to do this....

https://www.npmjs.com/package/vite-plugin-externals
